static/shell/unified_setup.sh

```diff
@@ -312,6 +312,12 @@ while [[ $# -gt 0 ]]; do
   esac
 done
 
+echo "Marking the nvidia information as Hold for apt package installer"
+major_version=$(nvidia-smi --query-gpu=driver_version --format=csv,noheader,nounits | head -n 1 | cut -d. -f1)
+
+sudo apt-mark hold nvidia-dkms-$major_version
+sudo apt-mark hold nvidia-driver-$major_version
+sudo apt-mark hold nvidia-utils-$major_version
```

Comment:
Why not doing this as first thing in the script?

Reply:
The above ones are functions and similar upgrade blocking calls + arg processing. I thought before we start installing anything do this should be enough. I can make it the very first thing but i don't that will make a difference.